arpeggio/tests/test_parsing_expressions.py

```diff
@@ -201,18 +199,17 @@ def grammar():
        "Expected 'b' at position (1, 7) => 'a, c, *'."
     ) == str(e.value)
 
-    # FIXME: This looks strange. Shouldn't be ',' at position 6?
-    with pytest.raises(NoMatch):
+    # FIXME: Shouldn't the ',' be at position 5?
```

Comment:
I think that the position is good as `,` should be found instead of `c` (at the position of `c`) and the column number is one-based.